notebooks/preprocess.ipynb

```diff
@@ -450,7 +450,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.8"
+   "version": "3.8.3"
```

Comment:
Is there a reason for this python version change?

Reply:
No this was not intentional